lib/language_pack/ruby.rb

```diff
@@ -603,6 +603,12 @@ def build_bundler(default_bundle_without)
           }
           env_vars["BUNDLER_LIB_PATH"] = "#{bundler_path}" if ruby_version.ruby_version == "1.8.7"
           puts "Running: #{bundle_command}"
+          # bundler binstub bug in Ruby 2.5.0-preview1
+          # https://bugs.ruby-lang.org/issues/13997
+          if ruby_version.ruby_version == "2.5.0"
```

Comment:
Should probably make this a ">="

Reply:
That's fair, I was hoping we can remove this when ruby 2.5.0 releases. Honestly, it also works for every version of ruby, so maybe we just scratch the `if`.